features/visitor_can_view_articles_in_categories.feature

```diff
@@ -4,14 +4,6 @@ Feature: Visitor can view articles in categories
     I would like to be able to filter the articles displayed in categories
 
     Background:
```

Comment:
Why are you removing the background step for creating the categories here?  it is not relevant for this PR.